lib/Backend/amd64/LowererMDArch.cpp

```diff
@@ -1643,6 +1653,12 @@ LowererMDArch::GeneratePrologueStackProbe(IR::Instr *entryInstr, IntConstType fr
     // $done:
     //
 
+    if (this->m_func->IsTrueLeaf() &&
```

Comment:
Sure.

---
In reply to: [69773959](https://github.com/Microsoft/ChakraCore/pull/1239#discussion_r69773959) [](ancestors = 69773959)